### DISCUSSION TITLE
VT not updating minisplit correctly

### DISCUSSION THREAD

**Discussion by AlbertShovelton** [jmcollin78/versatile_thermostat#477] [Q&A]
I have a bit of a weird setup, where I want to control two minisplit heads as one thermostat using VT. 90% of the time it works as expected, but I run into two issues.

1) when the mini splits are off and I turn them on through VT, both minisplit do not always turn on or turn on to the incorrect mode. I.e. turn on to Auto when they should be in cool. 

2) when I set the temperature on the VT thermostat card, the mini splits go to a different temperature. I.e I set the temperature to 74, the mini splits for to 72 or 76. I have self regulation off in the configuration, but it looks like it’s trying to self regulate based on room temperature.

**Comment by jmcollin78** [comment]
Hello, if you turn on the auto-regulation option "use internal temperature device" (something like that) the target temp can be different on each device.

**Comment by AlbertShovelton** [comment]
For now, I don’t want the target temperature to be different because I want
to confirm it’s working as expected. With auto-regulation off, I’d expect
units to go to the temperature I set it to, which doesn’t always happen.

**Comment by AlbertShovelton** [comment]
Thinking about it, the behavior seems like auto-regulation is on. But the
setting in the config clearly says no auto regulation. I am using the
central configuration, could that be it?

**Reply by jmcollin78** [reply]
no. Regulation is not a central configuration.
